woocommerce/woocommerce
# [e2e-utils-playwright] Add linting configuration and fix errors

### Changes proposed in this Pull Request:

Closes https://github.com/woocommerce/woocommerce/issues/52940

Configure linting for the newly added `e2e-utils-playwright` package.

- Added the lint pnpm script.
- Add the config.ci entry to execute the lint job in CI.
- Fixed the linting errors.

### How to test the changes in this Pull Request:

Locally, run the lint check:

```
pnpm --filter="@woocommerce/e2e-utils-playwright" lint
```

In CI, check the [Lint - @woocommerce/e2e-utils-playwright](https://github.com/woocommerce/woocommerce/actions/runs/11890863871/job/33130574989?pr=52664#logs) job.
